### PR TITLE
Update develop to 2.2.0

### PR DESCRIPTION
**Pull request by mattmiller87**

### CODE REVIEW

**Review by glennmatthews — COMMENTED** [review]
Probably need to run `towncrier build --version 2.1.1` to get the release-notes updated.

File: poetry.lock
Comment (glennmatthews) [inline poetry.lock:1]: What was the reason for regenerating this file? Shouldn't need to be touched as a part of the release prep.

**Comment by jvanderaa** [comment]
That towncrier part was new. I thought it was still `poetry version patch`. Will check on this.

**Comment by chadell** [comment]
We should do a minor version because we are adding new features not only patches

**Review by glennmatthews — APPROVED** [review]

**Pull request merged**